apps/src/lab2/levelEditors/aichatSettings/CollapsibleFieldSection.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+
+import {AiCustomizations} from '@cdo/apps/aichat/types';
+import CollapsibleSection from '@cdo/apps/templates/CollapsibleSection';
+
+import FieldSection from './FieldSection';
+
+import moduleStyles from './edit-aichat-settings.module.scss';
+
+interface CollapsibleFieldSectionProps {
```

Comment:
nit: we could export `FieldSectionProps` from `FieldSection` and have this extend that since it looks like it's the same, just with the additional `labelText`?